### DISCUSSION TITLE
Possible without SwiftUI?

### DISCUSSION THREAD

**Discussion by Elamuruga** [gonzalezreal/swift-markdown-ui#311] [Q&A]
In my project I am working on a storyboard and UIKIt is it possible to use the swift-markdown plugin for that? I want to achieve Github-flavoured markdown support. Could anyone help me with this? Thanks in advance.

**Comment by yccheok** [comment]
It is possible. I manage to integrate it into an UIKit app - https://github.com/gonzalezreal/swift-markdown-ui/discussions/366

For technical details, please kindly refer to https://github.com/gonzalezreal/swift-markdown-ui/issues/315
